AffirmSDK/AffirmCreditCard.m

```diff
@@ -48,12 +44,17 @@ - (instancetype)initWithDict:(NSDictionary *)dict
         }
         _checkoutToken = dict[@"checkout_token"];
         _created = dict[@"created"];
-        _cvv = dict[@"cvv"];
-        _number = dict[@"number"];
-        _callbackId = dict[@"callback_id"];
-        _cardholderName = dict[@"cardholder_name"];
-        _expiration = dict[@"expiration"];
         _creditCardId = dict[@"id"];
+        _callbackId = dict[@"callback_id"];
+        _cvv = nil;
+        _number = nil;
+        _cardholderName = nil;
+        _expiration = nil;
+        
+        if (dict[@"cvv"]) _cvv = dict[@"cvv"];
```

Comment:
Although this works, if these instance variables are nullable, you can just assign from the dict

Reply:
Thank you!

Reply:
@bf-affirm Fixed!